modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -285,4 +285,11 @@ routes:
 # ...
 ----
 <1> The static IP address for the Ethernet interface.
-<2> Next hop address for the node traffic. This must be in the same subnet as the IP address set for the Ethernet interface.
+<2> The next hop address for the node traffic. This must be in the same subnet as the IP address set for the Ethernet interface.
+
+[IMPORTANT]
+====
+You cannot use `br-ex` bridge, an OVNKubernetes-managed Open vSwitch bridge, as the next hop interface when configuring a static route unless you manually configured a customized `br-ex` bridge.
```

Comment:
`s/OVNKubernetes/OVN-Kubernetes/` ?  Whatever is the official name.

"use **the** `br-ex` bridge" maybe.  

You cannot use the OVN-Kubernetes `br-ex` bridge as the next hop interface ...

maybe.